openpype/hosts/webpublisher/plugins/publish/collect_published_files.py

```diff
@@ -174,6 +177,8 @@ def _get_family(self, settings, task_type, is_sequence, extension):
                 (family, [families], subset_template_name, tags) tuple
                 AssertionError if not matching family found
         """
+        if task_type:
```

Comment:
Wouldn't be better to lower all keys in `settings` and `task_type` instead if expecting that only one character is wrong?

Reply:
I agree, this would be much safer to work with

Reply:
agreed as well